phpBB/adm/style/ajax.js

```diff
@@ -429,12 +429,49 @@ $(function() {
 		$field_icon.after('<i style="font-size: 16px; margin:0 6px; vertical-align: middle;"></i>');
 	}
 
-	$('#field_icon').on('keyup blur', function() {
+	$field_icon.on('keyup blur', function() {
 		var input = $(this).val();
 		var $icon = $(this).next('i');
-		$icon.attr('class', 'o-icon o-icon-font fa-fw fa-' + input + ' fas');
+		$icon.attr('class', 'o-icon o-icon-font fa-fw fa-' + input + ' fas acp-icon');
```

Comment:
```
if (fieldIcon) {
	fieldIcon.addEventListener('keyup', updateIconClass);
	fieldIcon.addEventListener('blur', updateIconClass);
}

function updateIconClass(event) {
	const input = event.target.value;
	const icon = event.target.nextElementSibling;

	if (icon && icon.tagName.toLowerCase() === 'i') {
		icon.className = `o-icon o-icon-font fa-fw fa-${input} fas acp-icon`;
	}
}
```

Reply:
That works, but in case of empty icon name field leaves `fa-` in classes.